FBSnapshotTestCase.podspec

```diff
@@ -18,10 +17,5 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '6.0'
   s.requires_arc = true
   s.framework    = 'XCTest'
-
-  s.source_files = 'FBSnapshotTestCase.{h,m}', 'FBSnapshotTestController.{h,m}', 'UIImage+*{h,m}'
-
-  fb_def = 'FB_REFERENCE_IMAGE_DIR'
-  fb_val = '"$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages"'
-  s.prefix_header_contents = "#define #{fb_def} = #{fb_val}"
+  s.source_files = 'FBSnapshotTestCase/**/*.{h,m}'
```

Comment:
Tangential question: have we considered packaging ios-snapshot-test-case as an Xcode project? This would allow users to link it to their tests using methods other than CocoaPods, such as Carthage.